modules/prebid/rulesengine/bidder_config_ruleset_builder.go

```diff
@@ -0,0 +1,271 @@
+package rulesengine
+
+import (
+	"encoding/json"
+	"errors"
+	"slices"
+	"strings"
+
+	"github.com/prebid/prebid-server/v3/modules/prebid/rulesengine/config"
+	"github.com/prebid/prebid-server/v3/rules"
+)
+
+type CountryExclusions = map[string][]string
+type CountryGroups = map[string][]string
+
+// defaultCountryGroups maps country group names to their member countries
+var defaultCountryGroups = CountryGroups{
+	"EEA": []string{
+		"AND", "AUT", "BEL", "BGR", "CHE", "CYP", "CZE", "DEU", "DNK", "EST", "ESP",
+		"FIN", "FRA", "GBR", "GRC", "HRV", "HUN", "IRL", "ISL", "ITA", "LIE",
+		"LTU", "LUX", "LVA", "MLT", "NLD", "NOR", "POL", "PRT", "ROU",
+		"SWE", "SVN", "SVK",
+	},
```

Comment:
Don't we have a config for the EEA definition? `EEACountries`?

Reply:
I agree it makes more sense to use the config value here. I see though that the config `EEACountries` defaults to a list of 44 countries when there are actually only 30 in the EEA. Perhaps we need to change the default. I am not sure where the list of 44 countries came from.